docs/tutorials/inference/latent_distribution_test.ipynb

```diff
@@ -172,7 +175,7 @@
     "P2 = np.array([[0.9, 0.4],\n",
     "               [0.4, 0.9]])\n",
     "\n",
-    "A3 = sbm(csize, P2)\n",
+    "A3 = sbm(csize_2, P2)\n",
     "heatmap(A3, title='2-block SBM adjacency matrix A3')\n",
     "X3 = AdjacencySpectralEmbed(n_components=n_components).fit_transform(A3)\n",
     "_ = pairplot(X3, title='2-block adjacency spectral embedding A3', height=4.5)"
```

Comment:
how do you feel about plotting both the embeddings of A1 and A3 on the same plot using labels?

Reply:
Addressed in lines 181 and 184